src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+/**
+ * DukeException is a subclass of Exception that handles all the Duke exceptions.
+ */
+
+public class DukeException extends Exception {
+    private String msg;
+
+    /**
+     * Constructor for a duke exception
+     * @param msg a text to be printed out
+     */
+    public DukeException(String msg) {
+        this.msg = msg;
+    }
+
+    @Override
```

Comment:
You can actually use switch for this, it uses the .equals() function on strings, so you can consider that option!